src/plugins/bmqauthnbasic/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Basic Authenticator plugin
+# --------------------------
+
+if (NOT BMQ_TARGET_AUTHNBASIC_NEEDED )
+  return()
+endif()
+
+find_package( PkgConfig REQUIRED )
+
+bmq_add_plugin( bmqauthnbasic )
```

Comment:
The plugin should have `version` and `versiontag` components, to make sure that all static linking happens with the correct build.  See the prometheus plugin for an example.